src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,38 @@
+.loginCont{
+  display: flex;
+  flex-flow: column;
+  width: 400px;
+  height: 295px;
+  margin: 0 auto;
+  border: 1px solid black;
+  background-color: rgb(238, 238, 238);
+  align-items: center;
+
+}
+
+.inputField{
+  width: 60%;
+  height: 48px;
+  margin: 5px;
+  min-width: 200px;
+  border: 1px solid black;
+}
+
+.interactionsCont{
+  display: flex;
+  flex-flow: row;
+  width: 60%;
+  justify-content: space-between;
+}
+
+.loginButton{
```

Comment:
Different font-style for login button registration link and h2. Please make h2 and registration link adopt button's font style. Furthermore transform registration link. Make it's underline disappear and make it a little bit lighter blue.

Reply:
Fixed font styles, removed underline from registration link. Precise color of <a href> tag colour changes depending on the browser I believe.